### DISCUSSION TITLE
3rd party errors customisation

### DISCUSSION THREAD

**Discussion by McMerph** [RobinTail/express-zod-api#914] [Q&A] [answered]
How to customise 3rd party errors?
For example, if the user sends a request with invalid JSON, this will result in a response body with `status` and `error` fields.
But how to respond with plain text instead of JSON?
Is the workaround described in #907 the right way?

**Comment by RobinTail** [comment, marked as answer]
@McMerph ,

In your configuration you can specify a custom result handler for such errors. It can respond in a plain text. 

```ts
// custom ResultHandlerDefinition for common errors,
  // default: defaultResultHandler()
  errorHandler?: ResultHandlerDefinition<any, any>;
```

**Reply by RobinTail** [reply]
Please let me know of you tried it and it didn't work for some reason.

**Comment by McMerph** [comment]
It works! Thank you, @RobinTail !
